cmd/vclusterctl/cmd/platform/add/cluster.go

```diff
@@ -143,7 +143,7 @@ func (cmd *ClusterCmd) Run(ctx context.Context, args []string) error {
 
 	if os.Getenv("DEVELOPMENT") == "true" {
 		helmArgs = []string{
-			"upgrade", "--install", "loft", "./chart",
+			"upgrade", "--install", "loft", cmp.Or(os.Getenv("DEVELOPMENT_CHART_DIR"), "./chart"),
```

Comment:
This came in handy when testing with `go run cmd/vclusterctl/main.go` and wanting to point to a different chart directory.  Its not related to the aim of the PR.